### PR TITLE
 Add GitHub Profile to Badges category

### PR DESCRIPTION
**Pull request by Ileriayo**
## What type of PR is this? (check all applicable)

- [x] 🚀 Added Name
- [ ] ✨ Feature
- [ ] ✅ Joined Community
- [x] 🌟 ed the repo
- [ ] 🐛 Grammatical Error
- [x] 📝 Documentation Update
- [ ] 🚩 Other

## Description
Add GitHub Profile to Badges category

## Add Link of GitHub Profile
https://github.com/ileriayo/ileriayo

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -166,6 +168,7 @@
 - ["How to create a Github Profile README with Dynamic Github Stats"](https://codewithghazi.com/how-to-create-a-github-profile-readme-with-dynamic-github-stats/) - *Ghazi Khan*
 - ["How I Built A Self-Updating README On My Github Profile"](https://www.mokkapps.de/blog/how-i-built-a-self-updating-readme-on-my-git-hub-profile/) - *Michael Hoffmann (Mokkapps)*
 - ["Building a self-updating profile README for GitHub"](https://simonwillison.net/2020/Jul/10/self-updating-profile-readme/) - *Simon Willison*
+- [Markdown Badges](https://dev.to/ileriayo/mardown-badges-2og0)
```
Comment (abhisheknaiidu) [inline README.md:171]: Provide some description

File: README.md [hunk 2]
```diff
@@ -155,6 +156,7 @@
 - [Github Profile README Generator](https://github.com/rahuldkjain/github-profile-readme-generator) - This tool provides an easy way to create github profile readme with latest addons like `visitors count`, `github stats` etc.
 - [Dynamic Profile Page On Github](https://github.com/umutphp/github-action-dynamic-profile-page) - Get dynamically generated list of your commits (of the repositories that the action is configured) on GitHub profile readme.
 - [npm package downloads](https://github.com/maddhruv/github-readme-npm-downloads) - Show all of your npm packages and their total downloads
+- [Markdown Badges](https://github.com/Ileriayo/markdown-badges)
```
Comment (abhisheknaiidu) [inline README.md:159]: Provide description

Comment (Ileriayo) [reply README.md:159]: An opensource repo for tech badges suitable for markdown files.

File: README.md (hunk 1 above)
Comment (Ileriayo) [reply README.md:171]: A getting started article on markdown badges.

Comment (abhisheknaiidu) [reply README.md:171]: I mean provide description in the README

File: README.md (hunk 2 above)
Comment (abhisheknaiidu) [reply README.md:159]: I mean provide description in the README

File: README.md (hunk 1 above)
Comment (Ileriayo) [reply README.md]: Done.

**Comment by Ileriayo** [comment]
Resolved @abhisheknaiidu

File: README.md [hunk 3]
```diff
@@ -151,6 +151,7 @@
 
 #### Badges 🎫
 - [Anirudh Emmadi](https://github.com/aemmadi/aemmadi)
+- [Ileriayo Adebiyi](https://github.com/ileriayo/ileriayo)
```
Comment (abhisheknaiidu) [inline README.md:154]: Add it to the `last`

File: README.md [hunk 4]
```diff
@@ -178,6 +179,8 @@
 - [Github Profile README Generator](https://github.com/rahuldkjain/github-profile-readme-generator) - This tool provides an easy way to create github profile readme with latest addons like `visitors count`, `github stats` etc.
 - [Dynamic Profile Page On Github](https://github.com/umutphp/github-action-dynamic-profile-page) - Get dynamically generated list of your commits (of the repositories that the action is configured) on GitHub profile readme.
 - [npm package downloads](https://github.com/maddhruv/github-readme-npm-downloads) - Show all of your npm packages and their total downloads
+- [Markdown Badges](https://github.com/Ileriayo/markdown-badges) - Add badges to your profile.
+- [All Dev Stats in Readme](https://github.com/anmol098/waka-readme-stats) - [WakaTime](https://wakatime.com/) All Weekly Metrics on your Profile Readme with extra feature flags to toggle code Editors used, os worked on, show timezone, top working projects
```
Comment (abhisheknaiidu) [inline README.md:183]: Why are you adding this, remove it

Comment (Ileriayo) [reply README.md]: Done

File: README.md (hunk 3 above)
Comment (Ileriayo) [reply README.md]: Done.

**Pull request merged**